src/stratis_cli/_error_reporting.py

```diff
@@ -56,6 +56,15 @@ class StratisCliErrorCodes(IntEnum):
 )
 
 
+def exit_(code, msg):
+    """
+    Description
+    """
+    # if msg is not None
+    print(msg, file=sys.stderr)
```

Comment:
Take a look at the other place where we explicitly print. That invocation is very explicit with a bunch of parameters; it's probably best to be that way here, too.